storage/inspector/src/main.rs

```diff
@@ -44,7 +44,7 @@ enum Command {
 fn print_head(db: &LibraDB) -> Result<()> {
     let si = db
         .get_startup_info()
-        .expect("Can't get startup info")
+        .map_err(|_| anyhow!("Can't get startup info."))?
```

Comment:
what happens if we return an error here? this is also a utility app, I'm not sure I see the point in this modificatoin.